### PR TITLE
Fix Bower install in OSX - add quotes (")

### PR DESCRIPTION
**Pull request by ismaelfaro**
Also, in bower.json definition, I add quotes (") to end the "polymer" dependencies line.

### CODE REVIEW

**Comment by ismaelfaro** [comment]
I signed it!

File: 1.0/docs/start/getting-the-code.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ This generates a basic `bower.json` file. Some of the questions, like
 
 The next step is to install {{site.project_title}}:
 
-    bower install --save Polymer/polymer#^1.0.0
+    bower install --save "Polymer/polymer#^1.0.0"
```
Comment (arthurvr) [inline 1.0/docs/start/getting-the-code.md]: I agree with the other change but why did you add these? They're not required.

**Comment by arthurevans** [comment]
Thanks, @ismaelfaro (and good to meet you the other day).

File: 1.0/docs/start/getting-the-code.md (hunk 1 above)
Comment (arthurevans) [inline 1.0/docs/start/getting-the-code.md]: True, we don't use these on the command line.
